apps/chronicles/screens/components/PopoverScreen.tsx

```diff
@@ -10,37 +10,74 @@ import React, { useRef, useState } from "react";
+    const firstButtonRef = useRef<View>(null);
+    const [isFirstPopoverOpen, setIsFirstPopoverOpen] = useState(false);
+
+    const secondButtonRef = useRef<View>(null);
+    const [isSecondPopoverOpen, setIsSecondPopoverOpen] = useState(false);
+
     const style = useStyles(themeStyles);
     return (
         <ScrollView
             contentInsetAdjustmentBehavior="automatic"
             contentContainerStyle={style.contentContainer}
         >
             <Typography>
-                {
-                    "The popover component displays modally on top of your content and positions itself \
-                automatically based on constraints around the anchor element so you won't have to \
-                worry about it being positioned badly."
-                }
+                {"The popover component displays modally on top of your content and positions " +
+                    "itself automatically based on constraints around the anchor element so you " +
+                    "won't have to worry about it being positioned badly."}
             </Typography>
-            <Spacer amount="large" />
+            <Spacer />
             <Button
                 title="Trigger popover"
-                ref={buttonRef}
+                ref={firstButtonRef}
                 onPress={() => {
-                    setOpen(!open);
+                    setIsFirstPopoverOpen(!isFirstPopoverOpen);
                 }}
             />
             <Popover
-                open={open}
-                anchorEl={buttonRef}
-                onClose={() => setOpen(false)}
+                open={isFirstPopoverOpen}
+                anchorEl={firstButtonRef}
+                onClose={() => setIsFirstPopoverOpen(false)}
                 placement="top"
             >
                 <Typography>This is a popover</Typography>
             </Popover>
+
+            <Spacer amount="large" />
+            <Typography>
+                {"If you want to have the popover stay open while interacting " +
+                    "with other elements outside of it simply don't provide " +
+                    "an onClose callback. Though this might seem like convenient " +
+                    "functionality, please note that nothing prevents the user " +
+                    "from interacting with the rest of the app while the popover " +
+                    "is open, meaning that they can potentially end up in strange UI states."}
+            </Typography>
+            <Spacer />
+            <Button
+                title="My popover won't stop you from interacting.."
+                ref={secondButtonRef}
+                onPress={() => {
+                    setIsSecondPopoverOpen(!isFirstPopoverOpen);
+                }}
+            />
+            <Popover open={isSecondPopoverOpen} anchorEl={secondButtonRef} placement="top">
+                <Typography>{"But remember to provide a way to close it!"}</Typography>
```

Comment:
I see this text is inside curly brackets and quotes, while text some place above in this file does not have this. I prefer to not have it if it's not needed.